src/config/dynamic-theme-fixes.config

```diff
@@ -25785,7 +25785,7 @@ CSS
 
 ================================
 
-www.gamer.com.tw
+*.gamer.com.tw
```

Comment:
Changing www. to * would need to move that site fix up to have it sorted alphabetically.

I propose to leave first www.gamer.com.tw line and just add *. alias then it should work fine and moving won't be needed.